stylesheets/_component.panels.scss

```diff
@@ -19,6 +19,7 @@
 
     &__body {
         max-height: 20em;
```

Comment:
I know this is existing, but why do we need to specify a max-height here? Without it we could remove the overflow issue completely.

Reply:
Currently it's not desirable for the panel to exceed a set height when the body content is variable in length.